### PR TITLE
Release fixes

### PR DESCRIPTION
**Pull request by Cod3lta**
This PR fixes various bugs, tweaks and some TODOs we've had for a long time.

Here's a list of what have changed:
- Removed the "news" section from the home page until the `articles` is done
- Stats page now using Plan in a new tab
- Changed some texts on the homepage's about section
- versions and online players now gather data from real services
- Reworked the "staff" section on the home page
- Various CSS tweaks

Hopfully this PR announces the first release of the website! :tada:

### CODE REVIEW

**Review by lululombard — APPROVED** [review]
Looks good to me!

File: app/Helpers/DiscordHelper.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+{
+
+    public static function getPlayersConnected()
+    {
+        try {
+            $discordInfo = Cache::remember('discord-json-api', config('services.discord-json-api-time'), function () {
+                return Http::get(config('services.discord.json-api'))->json();
+            });
+        } catch (ConnectionException $e) {
+            return 0;
```
Comment (Nano112) [inline app/Helpers/DiscordHelper.php:19]: maybe return -1 to signifie a invalid number

File: app/Helpers/DiscordHelper.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        }
+
+        $members = $discordInfo['members'];
+        $membersCount = 1;
+        foreach ($members as $member) {
+            if ($member['status'] != 'offline') {
+                $membersCount++;
+            }
+        }
```
Comment (Nano112) [inline app/Helpers/DiscordHelper.php:28]: this could be cleaned up and generalised with array_count_values

**Review by Nano112 — CHANGES_REQUESTED** [review]
One small change

**Review by Nano112 — APPROVED** [review]

**Pull request merged**